### ISSUE TITLE
Team Logo discussion

### ISSUE THREAD

**Issue by annaciula** [rht-labs/cs-handbook#2]
this is a test to see how the discussion may look like for an item the doesn't exist so far. Question to all - do you think we need a team logo?
@mwalker5000 @shabnoorshah

**Comment by annaciula** [comment]
![bitmoji](https://sdk.bitmoji.com/render/panel/d38720bd-60aa-4e4d-91ee-cef1b0d5ac72-121ed7eb-1622-48d8-9a90-fa35914f6431-v1.png?transparent=1&palette=1&width=246)

**Comment by annaciula** [comment]
@mtakane i'm also testing this one for discussions

**Comment by kattekrab** [comment]
Yes! Having some kind of visual identity could help bust our silos. 

![image](https://user-images.githubusercontent.com/172755/199413046-d5248eb1-56b5-44bc-8cd2-e6fca6b90303.png)
